client/lib/feed-stream-store/index.js

```diff
@@ -76,10 +77,10 @@ function filterConversationsPosts( posts ) {
 	const newPosts = map( posts, this.keyMaker );
 	return filter( newPosts, post => {
 		// only keep things that we don't have or things that have new comments
-		if ( ! this.postById.has( post.postId ) ) {
+		if ( ! this.postById.has( keyToString( post ) ) ) {
 			return true; // new new
 		}
-		const existingPost = find( this.postKeys, { postId: post.postId } );
+		const existingPost = find( this.postKeys, postKey => keysAreEqual( postKey, post ) );
```

Comment:
ah. latent bug. i see. I wonder how often thats happened without anybody noticing.  it would only occur when hitting the pill, so....no way to know